timescaledb/getting-started/create-hypertable.md

```diff
@@ -1,28 +1,29 @@
-# 3.Create a hypertable
-When you've launched your first TimescaleDB instance and accessed your database,
-you can create your first hypertable. Hypertables are the heart of TimescaleDB
-and are what allows TimescaleDB to work so effectively with time-series data.
-
-## Chunks and hypertables
-Chunks and hypertables make storing and querying times-series data blazingly
-fast at peta-byte scale. TimescaleDB automatically partitions time-series data
-into chunks, or sub-tables, based on time and space. For example, chunks can be
-based on hash key, device ID, location or some other distinct key. You can
-configure chunk size so that recent chunks fit in memory for faster queries.
-
-A hypertable is an abstraction layer over chunks that hold time-series data.
-Hypertables allow you to query and access data from all the chunks as if they
-were in a single table. This is because commands issued to the hypertable are
-applied to all of the chunks that belong to that hypertable.
+# Create a hypertable
+Now that you've launched your first TimescaleDB instance and accessed your database,
+you can create your first hypertable. Hypertables are the heart of TimescaleDB functionality
+and allow TimescaleDB to work efficiently with time-series data.
+
+## Hypertables and chunks
+Hypertables and chunks make storing and querying times-series data fast at peta-byte 
+scale. 
+
+TimescaleDB automatically partitions time-series data into **chunks**, or sub-tables, 
+based on time and space. You can configure chunk size so that recent chunks fit in memory 
+for faster queries. 
+
+A **hypertable** is an abstraction layer over chunks that hold the time-series data.
+Hypertables allow you to query and access data from all the chunks within the hypertable. 
 
 Hypertables and chunks enable superior performance for shallow and wide queries,
 like those used in real-time monitoring. They are also good for deep and narrow
 queries, like those used in time-series analysis.
 
-<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/illustration-hypertable-chunk.png" alt="hypertable and chunks"/>
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/hypertables-chunks.png" alt="hypertable and chunks"/>
```

Comment:
I feel like it is a good place to quickly show the concept given that we talk about it briefly. @ryanbooz what are your thoughts?

Reply:
@ryanbooz ping

Reply:
So Ryan messaged me separately that he thinks it would be okay to add. Though he did suggest maybe I should add a more specified image showing this specific data and how it would be structured in the same way (like do this on the adding-data page??) what do you think about that idea @charislam?

Reply:
Yeah I think that's great!